[IMP] Sales: Aligning 'Optional products' page (#16686)
odoo/documentation: content/applications/sales/sales/send_quotations/optional_products.rst

Optional products
=================

The use of optional products is a marketing strategy that involves the cross-selling of useful and
related products alongside a desired core product. For instance, when a business configures optional
products in their Odoo database, an eCommerce or Website customer could be suggested a mouse and
keyboard or an extended warranty when they add a laptop to their shopping cart.

Optional products are automatically suggested during the quotation process whenever an associated
core product is added to a quote. They are also suggested in eCommerce interactions when a customer
adds an associated core product to their shopping cart.

.. note::
   Optional products differ from :doc:`accessory and alternative products
   </applications/websites/ecommerce/products/cross_upselling>` in terms of where they are displayed
   during customers' online shopping journeys.

Configuring optional products
=============================

With the Odoo *Sales* app, it is possible to add optional products directly to product forms. To add

> an optional product to a product form, navigate to :menuselection:`Sales --> Products --> Products`

Reviewer: ```suggestion
an optional product, navigate to :menuselection:`Sales --> Products --> Products`
```
We say "to a produc form" twice in a row, so since it is rigt after, it can be trimmed - and they will see it is the product form when they follow laong.

and choose a product.

Ensure that the product's :guilabel:`Can be Sold` checkbox is checked and click the
:guilabel:`Sales` tab. Under the :guilabel:`Upsell & Cross-sell` heading, the :guilabel:`Optional
Products` drop-down menu allows for optional products to be set. Products will be displayed in
alphabetical order. If the desired product isn't readily visible, type its name in the field to
bring it up, then select it to add it as an optional product.

To delete an optional product from the product form, simply click the :icon:`fa-times`
:guilabel:`(Delete)` icon.

.. image:: optional_products/optional-products-product-form.png
   :alt: The optional products section as it appears in product forms in Odoo Sales.

Additional products can also be added to a core product by clicking :guilabel:`Search more...`. This
opens the :guilabel:`Search: Optional Products` form, which displays all products in the catalog and
includes the :guilabel:`New` button to create a new product. Multiple products may be selected as
optional products at once when using this form by clicking their checkboxes and then clicking
:guilabel:`Select`.

> .. image:: optional_products/search-optional-products-form.png

Author: I think I'm actually going to cut the photo here. Something was telling me it was too basic to need to be laid out, and I should have gone with that instinct. Good call out!

   :alt: The Search: Optional Products form accessed by clicking Search more...

Optional products on quotations and quotation templates
=======================================================

> Quotation templates

Reviewer: I would recommend adding a small sentnece so that there is not a big heading and a sub head right under each other, stacked, with no infomration/sentence as the buffe.r I am personally aa BIG fan of anchors, and this would be a perfect opppurtinuity to do something like this:

Optional products can be added to quotation templates, allowing sales staff to offer related products without manually adding them to each quote. Additional optional products can also be added to individual quotations when needed.

The words "added to quotation templates" can link to an anchor point for the templaytes seciton, and the "individual quotations" text can link to the quotes section. This is nott required, but does provide a omre dynamic experience- I personally LOVE it when I can click on exactly what I want and jump right to it.

-------------------

Quotation templates also have an :guilabel:`Optional Products` tab, where related products or
services can be added. To add optional products to a quotation template, navigate to
:menuselection:`Sales app --> Configuration --> Quotation Templates`. Then, either select an

> existing quotation template to edit, or create a new one by clicking :guilabel:`New`.

Author: Good call out!

On the quotation template form, click the :guilabel:`Optional Products` tab. Under the
:guilabel:`Optional Products` tab, click :guilabel:`Add a line` and select the desired product to
add as an optional product to the quotation template.

.. image:: optional_products/optional-products-tab-quotation-template.png
   :alt: The Optional Products section in a quotation template in Odoo Sales.

When the configured quotation template is used, the products added in the :guilabel:`Optional
Products` tab will appear in the corresponding tab in the quotation. These products can be removed
and additional products can be added before the quotation is sent to a customer.

.. note::
   There is no limit to how many optional products can be added to a quotation template.

Quotations
----------

To add optional products directly to a quotation, click :guilabel:`Add a product` in the

> :guilabel:`Optional Products` tab of a quotation. Doing so reveals a blank field in the

Reviewer: This sort of jumps right in, but doesn't say where the quopte is created. I woudl just add a little  omre info, such as:

To add additional products on an individual quote, navigate to (Sales > Orders > Quotations). Select an existing quote, or create a new quote.

In the tet above, I woudl link "create a new uote" to the odcumentation that explains how to do that - this way, they have EVERYTHING they could possibly need. 

You coudl also add a note after my suggested lines that explains when you CANOT add items ot a quote (example, ifthey are cancelled?) I am not sur,e but if you know, you can add a little but of info liek that! Perhaps it'sb ased on the status?)

Reviewer: I forgot to add - we typically tell people where to go firs,t then the action to do. So in this cas, you would tell them to click into the tab, THEN click the text. It's a small change but when you stick to that order it makes the doc easier to follow as a reader.

:guilabel:`Product` column.

When clicked, a drop-down menu with products from the database appear. Select the desired product

> from the drop-down menu to add it as an optional product. Type the name of the desired product or

Reviewer: 2 things - the GUI isn't rendering properly (there's a character out of whack) but also, refer to my suggestion above about how to add more items.)

click :guilabel: `Search More...` to find additional products.

.. note::
   When a product is added, the default :guilabel:`Quantity` is `1`, but that can be edited at any

>    time.

Reviewer: ```suggestion
   When a product is added, the default :guilabel:`Quantity` is `1`, but can be updated.
```

To delete any line item from the :guilabel:`Optional Products` tab, click the :icon:`fa-trash-o`
:guilabel:`(delete)` icon.

Previewing optional products
============================

Click the :guilabel:`Preview` button, located in the upper-left corner of the quotation, to reveal a
preview of the quotation email customers would receive. Optional products are located in the
:guilabel:`Options` section of the preview.

.. image:: optional_products/optional-products-preview-button.png
   :alt: Preview your quotations on Odoo Sales.

Customers are able to add different optional products to an order by clicking the
:icon:`fa-shopping-cart` :guilabel:`(Add to cart)` icon, located to the right of the optional
product line.

> .. image:: optional_products/optional-products-options-section.png

Reviewer: The text in this screen is cut off, so I recommend ensuring everything is visible. Even if it'snot needed, it's just visually jarring to see things missing like that.  I actually recommend expanging what is shown, so they get more of a sense of what they are looking at:

For example:

<img width="719" height="512" alt="Image" src="https://github.com/user-attachments/assets/31fc259e-4600-42f1-81c9-4237bbb45c79" />

(But not htese products I selected - they make no sense, I was just adding filler)

   :alt: The Options section within a quotation preview.

> If a customer selects optional products, these are automatically added to the quotation managed by

Reviewer: ```suggestion
If a customer selects optional products, they are automatically added to the quotation managed by
```

Optional products
=================

The use of optional products is a marketing strategy that involves the cross-selling of useful and
related products alongside a desired core product. For instance, when a business configures optional
products in their Odoo database, an eCommerce or Website customer could be suggested a mouse and

> keyboard or an extended warranty when they add a laptop to their shopping cart.

Reviewer: ```suggestion
related products alongside a desired core product. For instance,  a business can recommend a mouse and keyboard when a customer adds a laptop to their shopping cart.
```
This is far more ocncise - I was getting lost in the original copy. Also, no one likes those 'extended warranty' calls, so I figured let's keep it to 2 products =D
